### PR TITLE
LPS-90645 Blog content out of range in the portlet

### PR DESCRIPTION
**Pull request by knchau**
Previous pr: https://github.com/blzaugg/liferay-portal/pull/215

Changed various asset content's style to break word in content according to https://github.com/blzaugg/liferay-portal/pull/215#issuecomment-487800818

https://issues.liferay.com/browse/LPS-90645

### CODE REVIEW

**Comment by knchau** [comment]
ci:test:sf

**Comment by knchau** [comment]
ci:test:relevant

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
 @import "mixins";
 
 .portlet-asset-publisher {
+	word-wrap: break-word;
+
 	.asset-full-content {
```
Comment (blzaugg) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss]: Move this under `.asset-full-content`

Comment (knchau) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss]: I originally tried putting it under there but it didn't change the wrapping style, so I placed it in its parent.

Comment (blzaugg) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss]: I believe `portlet-asset-publisher` is going to be too general, and apply to additional elements we are not intending. Can you see if there's another common ancestor under `.portlet-asset-publisher` that would be a good fit? Maybe something like `.asset-content`.

Comment (knchau) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss]: I understand, I'll look around!

Comment (knchau) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss]: I couldn't find any common ancestor under `.portlet-asset-publisher` that applies the fix. I also searched the project for `asset-content` and `asset-summary`, but there wasn't any relevant findings that would apply the fix. Would it be okay to create `.asset-content` as a child of `.portlet-asset-publisher` and apply the word-wrap property within there?

Comment (blzaugg) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss]: Can you paste DOM between `<div class="portlet-asset-publisher">` and the first content you want to style?

```html
<div class="portlet-asset-publisher">
    <div class="some-ancestor with-some-class">
        <div class="asset-summary">
            <div class="asset-content">
                <p>My content HTML</p>
```

Comment (knchau) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss]: The highlighted blue to the "ccc...".
dom_asset

**Comment by blzaugg** [comment]
Targeting `asset-content` should work just fine in this case.

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (blzaugg) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss]: Targeting `.asset-content` should work just fine in this case.

So you could use
```css
.asset-content, .asset-full-content {
```

**Comment by knchau** [comment]
ci:test:relevant

**Comment by knchau** [comment]
ci:test:sf

**Comment by gregory-bretall** [comment]
@knchau for reference, since this commit touches multiple components and the fixes are not dependent on each other, we will unfortunately have to send them as separate PRs on different LPS tickets for each individual component. This is because each component team has their own individual reviews process and UI standards. Let me know if you need any help breaking up the commit into multiple different tickets. Feel free to resend them directly to your normal reviewer once you've done so

**Pull request closed without merging**